inappmessaging/src/main/res/layout/message_image_view.xml

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.facebook.drawee.view.SimpleDraweeView xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  xmlns:fresco="http://schemas.android.com/apk/res-auto"
-  android:id="@+id/message_image_view"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:contentDescription="@string/in_app_message_image"
-  fresco:actualImageScaleType="fitCenter"
-  android:visibility="gone"
-  tools:visibility="visible"/>
\ No newline at end of file
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/message_image_view"
+    android:layout_width="match_parent"
```

Comment:
Are `layout_width` and `layout_height` responsible for maintaining correct aspect ratio of image?

Reply:
It's handled in the Picasso call side.